src/main/java/com/zup/pizzaria/dtos/ClienteDTO.java

```diff
@@ -11,6 +11,15 @@ public ClienteDTO(Long id, String nome, String email) {
         this.email = email;
     }
 
+    public ClienteDTO(String nome, String email){
+        this.nome = nome;
+        this.email = email;
+    }
+
+    public ClienteDTO(){
+
+    }
```

Comment:
Resumir construtores vazios em uma única linha tornaria o codigo mais limpo, acha que faz sentido?

```suggestion
    public ClienteDTO(){}
```

Reply:
Oi Laura, obrigada pela dica! Vou alterar aqui.